src/components/CodeAssignmentModal/BulkAssignFields.jsx

```diff
@@ -7,7 +7,7 @@ import { normalizeFileUpload } from '../../utils';
 
 const BulkAssignFields = () => (
   <>
-    <h3 className="mb-2">Add Users</h3>
+    <h3 className="mb-2">Add Learners</h3>
```

Comment:
Sentence case! (I know it wasn't originally, but that's the standard now, so we should update when we touch things.